Firestore/core/src/model/base_path.h

```diff
@@ -174,6 +184,31 @@ class BasePath {
 
  private:
   SegmentsT segments_;
+
+  static util::ComparisonResult compareSegments(const std::string& lhs,
```

Comment:
C++ [style](https://google.github.io/styleguide/cppguide.html#Function_Names) guide suggests function names to start with a capital letter (`CompareSegments`)